doc/source/writers.rst

```diff
@@ -136,3 +136,34 @@ and save them all at once.
     ...                          compute=False)
     >>> results = [res1, res2]
     >>> compute_writer_results(results)
+
+
+Adding text to images
+=====================
+
+Satpy, via `pydecorate`, can add text to images when they're being saved.
```

Comment:
Any idea if this renders as a link to pydecorate's documentation? I think you could make it

```
:doc:`pydecorate <pydecorate:index>`
```

And it would work.

Reply:
Done!